### PR TITLE
Update arrays.xml

### PR DESCRIPTION
**Pull request by essentialols**
Updated the playback speeds for voice messages to also include a 1.5x option. This will significantly improve the user experience based on my own experience, the experience of friends, and based on the popularity of this option in other similar apps.

I couldn't test it myself because my Android Studio won't work (I'm not an android developer). If somebody else could give this a try, it'd be greatly appreciated.

### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [ ] I have tested my contribution on these devices:
 * Device A, Android X.Y.Z
 * Device B, Android Z.Y
 * Virtual device W, Android Y.Y.Z
- [ ] My contribution is fully baked and ready to be merged as is
- [ ] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### CODE REVIEW

**Comment by essentialols** [comment]
I keep receiving error messages when trying to build the app in Android Studio. Perhaps somebody else could do it and confirm that it's working. Thanks!

**Comment by Caerbannog** [comment]
No issues has been created on Github, but the change has been requested on the forums:
https://community.signalusers.org/t/speeding-up-voice-memo-clips/2616/27?u=caerbannog

As of now, 6 users have expressed this need on the forum.

**Comment by cody-signal** [comment]
Thanks, pulled this in and will go out soon.

File: app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -406,11 +406,13 @@
 
     <integer-array name="PlaybackSpeedToggleTextView__speeds">
         <item>100</item>
+        <item>150</item>
         <item>200</item>
         <item>50</item>
     </integer-array>
     <string-array name="PlaybackSpeedToggleTextView__speed_labels">
         <item>@string/PlaybackSpeedToggleTextView__1x</item>
+        <item>@string/PlaybackSpeedToggleTextView__1p5x<item>
```
Comment (alex-signal) [inline app/src/main/res/values/arrays.xml:415]: Tag not properly closed, trailing <item> should be </item>

Also, you'll need to add the proper string resource.

Comment (essentialols) [reply app/src/main/res/values/arrays.xml:415]: Not sure how to append this pull request. I've created a new one that addresses the two issues. Here is the updated pull request: #11622

**Pull request closed without merging**